tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -0,0 +1,1378 @@
+    if isinstance(tvm_output, tvm.nd.NDArray):
+        tvm_output = [tvm_output]
+
+    assert len(tvm_output) == len(ort_output), "Unequal number of outputs"
+
+    for (tvm_out, ort_out) in zip(tvm_output, ort_output):
+        # TODO Allow configurable tolerance.
+        # Sometimes None is used to indicate an unused output.
+        if ort_out is not None:
+            tvm.testing.assert_allclose(tvm_out.numpy(), ort_out, atol=1e-5)
+
+
+@pytest.mark.parametrize("dynamic", [True, False])
+def test_matmul(dynamic):
+    matmul_node = helper.make_node("MatMul", ["a", "b"], ["c"])
+
+    tensor_size = [32, 32]
+    if dynamic:
+        tensor_size = ["?", "?"]
+
+    graph = helper.make_graph(
+        [matmul_node],
+        "matmul_test",
+        inputs=[
+            helper.make_tensor_value_info("a", TensorProto.FLOAT, tensor_size),
+            helper.make_tensor_value_info("b", TensorProto.FLOAT, tensor_size),
+        ],
+        outputs=[helper.make_tensor_value_info("c", TensorProto.FLOAT, tensor_size)],
+    )
+
+    model = helper.make_model(graph, producer_name="matmul_test")
+    inputs = None
+    if dynamic:
+        inputs = {
+            "a": np.random.normal(size=(32, 48)).astype("float32"),
+            "b": np.random.normal(size=(48, 64)).astype("float32"),
+        }
+    check_correctness(model, inputs)
+
+
+def test_concat():
+    concat_node = helper.make_node("Concat", ["a", "b"], ["ab"], axis=0)
+
+    graph = helper.make_graph(
+        [concat_node],
+        "concat_test",
+        inputs=[
+            helper.make_tensor_value_info("a", TensorProto.FLOAT, [1, 32]),
+            helper.make_tensor_value_info("b", TensorProto.FLOAT, [1, 32]),
+        ],
+        outputs=[helper.make_tensor_value_info("ab", TensorProto.FLOAT, [2, 32])],
+    )
+
+    model = helper.make_model(graph, producer_name="concat_test")
+    check_correctness(model)
+
+
+def test_add():
+    add_node = helper.make_node("Add", ["a", "b"], ["ab"])
+
+    graph = helper.make_graph(
+        [add_node],
+        "add_test",
+        inputs=[
+            helper.make_tensor_value_info("a", TensorProto.FLOAT, [1, 32]),
+            helper.make_tensor_value_info("b", TensorProto.FLOAT, [1, 32]),
+        ],
+        outputs=[helper.make_tensor_value_info("ab", TensorProto.FLOAT, [1, 32])],
+    )
+
+    model = helper.make_model(graph, producer_name="add_test")
+    check_correctness(model)
+
+
+def test_mul():
+    mul_node = helper.make_node("Mul", ["a", "b"], ["ab"])
+
+    graph = helper.make_graph(
+        [mul_node],
+        "mul_test",
+        inputs=[
+            helper.make_tensor_value_info("a", TensorProto.FLOAT, [1, 32]),
+            helper.make_tensor_value_info("b", TensorProto.FLOAT, [1, 32]),
+        ],
+        outputs=[helper.make_tensor_value_info("ab", TensorProto.FLOAT, [1, 32])],
+    )
+
+    model = helper.make_model(graph, producer_name="mul_test")
+    check_correctness(model)
+
+
+def test_cast():
```

Comment:
Are there more cases that might be of interest?

Reply:
I've parametrized this test